plugins/chat/assets/stylesheets/common/index.scss

```diff
@@ -0,0 +1,45 @@
+@import "base-common";
```

Comment:
Using only "common" wasn't working. I didn't dig much why as these files will be gone over time anyways.

Reply:
Looks like using "common" will import core's `common.scss` file. The same kind of clash will happen if core introduces any other identically-named stylesheets.

I was going to suggest using a relative import like `@import "./common";`, but it seems that plugin stylesheets are not properly namespaced during compilation so the same clash still happens 🤦‍♂️. A problem for another day I guess.